### PR TITLE
adds vesctl to devcontainer

### LINKED ISSUE #28: add vesctl to devcontainer
https://gitlab.com/volterra.io/vesctl

### PR DESCRIPTION
**Pull request by vinnie357**
closes #28

### CODE REVIEW

**Review by jmcalalang — APPROVED** [review]
I'll test this later today, we might need to update the aws-kic readme, but ill know more after trying it.

way to start and bring in volterra!

File: solutions/delivery/application_delivery_controller/nginx/kic/aws/admin.auto.tfvars.example [hunk 1]
```diff
@@ -1,3 +1,5 @@
+adminAccountName = "zadmin"
+resourceOwner = "yourname"
 awsRegion = "us-east-2"
 awsAz1    = "us-east-2a"
```
Comment (yossi-r) [inline solutions/delivery/application_delivery_controller/nginx/kic/aws/admin.auto.tfvars.example:4]: remove the az's, not all az's are available in every region so the user needs to know that in advance.
the network module dynamically chooses the available ones

File: solutions/delivery/application_delivery_controller/nginx/kic/aws/admin.auto.tfvars.example [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
 awsAz2    = "us-east-2b"
```
Comment (yossi-r) [inline solutions/delivery/application_delivery_controller/nginx/kic/aws/admin.auto.tfvars.example:5]: remove the az's, not all az's are available in every region so the user needs to know that in advance.
the network module dynamically chooses the available one

File: solutions/delivery/application_delivery_controller/nginx/kic/aws/main.tf [hunk 3]
```diff
@@ -5,8 +5,7 @@ provider "aws" {
 // Network
 module "aws_network" {
```
Comment (yossi-r) [inline solutions/delivery/application_delivery_controller/nginx/kic/aws/main.tf:6]: we should pass 'resourceOwner' to the network module

File: solutions/delivery/application_delivery_controller/nginx/kic/aws/main.tf [hunk 4]
```diff
@@ -47,14 +46,14 @@ module "eks" {
 
 // jumphost
 resource "aws_key_pair" "deployer" {
-  key_name   = "${var.userId}-${var.projectPrefix}"
+  key_name   = "${var.adminAccountName}-${var.projectPrefix}"
```
Comment (yossi-r) [inline solutions/delivery/application_delivery_controller/nginx/kic/aws/main.tf:49]: keyName should be from 'resourceOwner', not adminAccountName

File: solutions/delivery/application_delivery_controller/nginx/kic/aws/main.tf [hunk 5]
```diff
@@ -47,14 +46,14 @@ module "eks" {
...
   public_key = var.sshPublicKey
 }
 
 module "jumphost" {
```
Comment (yossi-r) [inline solutions/delivery/application_delivery_controller/nginx/kic/aws/main.tf:53]: we should pass 'resourceOwner' to the workstation module

**Review by yossi-r — APPROVED** [review]

**Pull request merged**